docs/apache-hive-catalog.md

```diff
@@ -140,15 +140,7 @@ Since 0.6.0-incubating, the data types other than listed above are mapped to Gra
 Table properties supply or set metadata for the underlying Hive tables.
 The following table lists predefined table properties for a Hive table. Additionally, you can define your own key-value pair properties and transmit them to the underlying Hive database.
 
-| Property Name      | Description                                                                                                                                | Default Value                                                                                                                                       | Required | Since version |
```

Comment:
how about adding columns of `reserved` and `immutable` to the table?

Reply:
I don't know Pimon's Since version. Could you clarify?

Reply:
0.6.0-incubating